.changeset/wise-owls-hug.md

```diff
@@ -0,0 +1,6 @@
+---
+'@redux-saga/core': patch
+'redux-saga': patch
+---
+
+Added warnings when using take(channelOrPattern) with more than one parameter
```

Comment:
```suggestion
Added warnings when using `take(channelOrPattern)` incorrectly with more than one parameter. It helps to surface problem with `take(ACTION_A, ACTION_B)` being used instead of `take([ACTION_A, ACTION_B])`.
```